### PR TITLE
[CSS] Disable completions in embedded template tags

### PR DESCRIPTION
**Pull request by deathaxe**
This PR suggests to modify CSS completion selector to prevent property completions in comments and embedded template code.

Without this restriction, 

1. template language related completions are often nearly completely hidden.
2. chances are high to commit property completions within template expressions.

... both of which degrates developer experience with regards to completions.

#### CSS

```
div {
  /* |<- no css property completions here */
}
```

#### CSS (PHP)

```
div {
   <?php |<- no css property completions here ?>
}
```

#### CSS (Jinja2)

```
div {
   {% |<- no css property completions here %}
   {{ |<- no css property completions here }}
}
```

same in all the others ...

Note: This PR already excludes `meta.selector` as it locally builds on top of #3983.

### CODE REVIEW

**Review by keith-hall — DISMISSED** [review]

**Comment by deathaxe** [comment]
Resolved conflicts with merged #3983.

**Comment by michaelblyons** [comment]
Is there a way to test the negated completions? If not, I suggest adding an extra negation (`- source.css source`) in case someone does not follow the `meta.embedded` convention.

**Comment by deathaxe** [comment]
We'd need some kind of unittest to check whether css_completion plugin returns something based on given cursor positions. Uncertain whether we could applicate UnitTesting to do that or if we want to.

Actually `- source.css source - source.css text` was my first choice either. Was uncertein though whether all template syntaxes apply `source` to their template tags. Maybe having both is foolprove.

**Review by michaelblyons — APPROVED** [review]

File: CSS/CSS.sublime-settings [hunk 1]
```diff
@@ -4,11 +4,11 @@
 
     // Controls what scopes default completions will be provided in.
     // Can be a list of strings which are joined before matching.
-    "default_completions_selector": "source.css",
+    "default_completions_selector": "source.css - source.css comment - source.css meta.embedded - source.css source - source.css text",
```
Comment (keith-hall) [inline CSS/CSS.sublime-settings:7]: I'll need to refresh my memory of multiple negative selector operator precedence to understand what this is doing...
A comment which explains this new default could help :)

Comment (deathaxe) [reply CSS/CSS.sublime-settings:9]: A reminder was added to settings.

We could also wrap all negative selectors into parens, if that helps improving readability, just like

```
"default_completions_selector": "source.css - (source.css comment | source.css meta.embedded | source.css source | source.css text)",
```

The idea is to exclude CSS comments and all sorts of embedded source/text under source.css.

Current selectors disable completions in `source.css source.php` for instance, but not in `source.php source.css`. Not quite sure it is a realistic example but selectors should just make sure to only exclude stuff under CSS.

**Review by keith-hall — APPROVED** [review]

**Review by michaelblyons — APPROVED** [review]

**Comment by deathaxe** [comment]
Thanks!

**Pull request merged**